### DISCUSSION TITLE
Mongoose doesn't work.

### DISCUSSION THREAD

**Discussion by bnsx** [oven-sh/bun#1104] [General]
How to fix this problem?
![image](https://user-images.githubusercontent.com/93707746/185622619-d84ac405-9363-4daa-836b-2a22fe0beb5c.png)

**Comment by Bellisario** [comment]
Looking into the source code, seems Bun doesn't have a built-in dns module, maybe it needs still to be implemented...

P.S.: it's the first time I see it :face_with_spiral_eyes:, maybe no one considered it, yet, anyway, seems it's a really [built-in Node module](https://nodejs.org/docs/latest-v16.x/api/dns.html#dns).

---

cc @Jarred-Sumner
